did-configuration/demo/scripts/create_did_configuration.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs");
+const path = require("path");
+const ES256K = require("@transmute/es256k-jws-ts");
+
+// feel free to impersonate me :/
+const privateJWK = {
+  crv: "secp256k1",
+  d: "rhYFsBPF9q3-uZThy7B3c4LDF_8wnozFUAEm5LLC4Zw",
+  kid: "JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
+  kty: "EC",
+  x: "dWCvM4fTdeM0KmloF57zxtBPXTOythHPMm1HCLrdd3A",
+  y: "36uMVGM7hnw-N6GnjFcihWE3SkrhMLzzLCdPMXPEXlA"
+};
+
+const publicJWK = {
+  crv: "secp256k1",
+  kid: "JUvpllMEYUZ2joO59UNui_XYDqxVqiFLLAJ8klWuPBw",
+  kty: "EC",
+  x: "dWCvM4fTdeM0KmloF57zxtBPXTOythHPMm1HCLrdd3A",
+  y: "36uMVGM7hnw-N6GnjFcihWE3SkrhMLzzLCdPMXPEXlA"
+};
+
+const defaultExpiresInHours = 99999;
+
+(async () => {
+  const innerPayload = {
+    iss: "did:btcr:xxcl-lzpq-q83a-0d5",
+    domain: "example.com",
+    exp: Math.floor(Date.now() / 1000) + 60 * 60 * defaultExpiresInHours
```

Comment:
default expiration is 1 hour for this library, I like that a DID controller can control when their domain claim will expire independent of the domain admin.